rebasehelper/utils.py

```diff
@@ -323,7 +322,26 @@ def close(self):
         session = requests.Session()
         session.mount('ftp://', FTPAdapter())
 
-        r = session.get(url, stream=True)
+        try:
+            return session.get(url, **kwargs)
+        except requests.exceptions.ConnectionError as e:
+            logger.debug('ConnectionError: %s', str(e))
```

Comment:
It's safer to use `six.text_type()` instead of `str()`, as it returns unicode string on python2.

Reply:
At this point you can easily catch other types of exceptions `requests.get()` can raise, so it would be nice to do so.

Reply:
So I should also catch `Timeout` and `TooManyRedirects`. `HTTPError` is taken care of in other parts of the code by checking the status code.

Reply:
Sure, that makes sense.